internal/backend/gs/config.go

```diff
@@ -16,13 +16,19 @@ type Config struct {
 	Bucket    string
 	Prefix    string
 
-	Connections uint `option:"connections" help:"set a limit for the number of concurrent connections (default: 5)"`
+	ConnectionsLimit uint `option:"connections_limit" help:"set a limit for the number of concurrent connections (default: 5)"`
```

Comment:
That parameter is called `connections` everywhere else. Please don't rename it.

Reply:
Reverted.